ethereum_history_api/circuits/lib/src/verifiers/account.nr

```diff
@@ -18,11 +18,12 @@ pub(crate) fn assert_account_equals(account_rlp_left_padded: [u8; MAX_ACCOUNT_ST
     let account_rlp_list: RlpList<ACCOUNT_FIELDS_COUNT> = decode_list_of_small_strings(account_rlp_right_padded);
     assert(account_rlp_list.len == ACCOUNT_FIELDS_COUNT, "Invalid number of fields in account RLP");
 
-    let balance = U128::from_integer(account.balance).to_be_bytes();
-    let balance = byte_value(balance);
-
     account_rlp_list.get(NONCE_INDEX).assert_eq_u64("Nonce", account_rlp_right_padded, account.nonce as u64);
-    account_rlp_list.get(BALANCE_INDEX).assert_eq_string("Balance", account_rlp_right_padded, balance);
+    account_rlp_list.get(BALANCE_INDEX).assert_eq_u128(
+        "Balance",
+        account_rlp_right_padded,
+        U128::from_integer(account.balance)
+    );
```

Comment:
Why not `.assert_equal_field()`?

Reply:
I think we don't want to create this function for field, since we wanted to remove these Field types. As for now balance is left as Field because of difficulties with getting U128 from oracles, but hopefully it will be changed and balance will be of U128 type.